lib/internal/modules/esm/loader.js

```diff
@@ -2,8 +2,7 @@
 
 const {
   FunctionPrototypeBind,
-  ObjectSetPrototypeOf,
-  SafeMap,
+  ObjectSetPrototypeOf
```

Comment:
```suggestion
  ObjectSetPrototypeOf,
```
not sure if this matters, but it had a trailing comma before

Reply:
The general style seems to be to not have trailing commas for object literals in the codebase, but I'm not sure if we have been consistent about this either.